web/internal/web_test.sh.template

```diff
@@ -38,4 +38,29 @@ fi
 
 printenv
 
-"$TEST_SRCDIR/%TEMPLATED_launcher%" --metadata="%TEMPLATED_metadata%" --test="%TEMPLATED_test%"
+args=()
+portatnext=0
+debuggerport=0
+
+for var in $@; do
+  if [[ $portatnext > 0 ]]; then
```

Comment:
This is a lexicographic comparison. For numeric comparison, use `(( 1 > 0 ))`.

Even better, make clear that $portatnext is a boolean -- use 1 for true and 0 for false, and evaluate it as `if (( "${portatnext}" )); then`.

For all comments, you can omit the braces when quoting variables if you want, solely for the sake of consistency with the existing code in this file.